### ISSUE TITLE
It looks like YOUR network has been temporary banned because of too many requests. 

### ISSUE THREAD

**Issue by blackseabreathe** [jsanahuja/jquery.instagramFeed#95]
console error is "Instagram Feed: It looks like YOUR network has been temporary banned because of too many requests. See [https://github.com/jsanahuja/jquery.instagramFeed/issues/25](https://github.com/jsanahuja/jquery.instagramFeed/issues/25)"

**latest 1.4.1 jQuery version, 61 string:**
`return void g.on_error("Instagram Feed: It looks like YOUR network has been temporary banned because of too many requests. See https://github.com/jsanahuja/jquery.instagramFeed/issues/25", 4);`
**It doesn't work with username.**

### How to fix it?
it happens every day, the error tells me that I'm banned, but why? to many requests?
**Note:** If I reload the page I'm using instagamFeed 20-30 times then I'll get this error until next day. Next day same problem after 20-30 times of reloading;
when I got this message nothing shows inside **div#instafeed**

**Your demo page has same error**

**Comment by blackseabreathe** [comment]
meanwhile **'tag'** works normal,

**my code as simple as it can be:**

```
(function ($) {
$(window).on('load', function () {
$.instagramFeed({
'username': 'instagram',
'container': "#instafeed",
'display_profile': false,
'display_biography': false,
'display_gallery': true,
'styling': false,
'items': 12,
'items_per_row': 6,
'margin': 0,
'image_size': 150
});
});
})(jQuery);
```

**Comment by blackseabreathe** [comment]
> console error is "Instagram Feed: It looks like YOUR network has been temporary banned because of too many requests. See #25"
> 
> **last jQuery version, 61 string:**
> `return void g.on_error("Instagram Feed: It looks like YOUR network has been temporary banned because of too many requests. See https://github.com/jsanahuja/jquery.instagramFeed/issues/25", 4);`
> 
> How to fix it?
> it happens every day, the error tells me that I'm banned, but why? to many requests?
> **Note:** If I reload the page I'm using instagamFeed 20-30 times then I'll get this error until next day. Next day same problem after 20-30 times of reloading;
> when I got this message nothing shows inside **div#instafeed**
> 
> **Your demo page has same error**

**Comment by lolyrycs** [comment]
if you put this jquery script in all the pages it will easly reach the request limit. (this was my case)
I tried to condition the script to the existance of the container, it limits the number of requests. 
Not a solution but hope it helps.


```
(function ($) {
$(window).on('load', function () {
if  ($('#instafeed').length) {
$.instagramFeed({
'username': 'instagram',
'container': "#instafeed",
'display_profile': false,
'display_biography': false,
'display_gallery': true,
'styling': false,
'items': 12,
'items_per_row': 6,
'margin': 0,
'image_size': 150
});
});
}
})(jQuery);
```

**Comment by blackseabreathe** [comment]
> if you put this jquery script in all the pages it will easly reach the request limit. (this was my case)
> I tried to condition the script to the existance of the container, it limits the number of requests.
> Not a solution but hope it helps.
> 
> ```
> (function ($) {
> $(window).on('load', function () {
> if  ($('#instafeed').length) {
> $.instagramFeed({
> 'username': 'instagram',
> 'container': "#instafeed",
> 'display_profile': false,
> 'display_biography': false,
> 'display_gallery': true,
> 'styling': false,
> 'items': 12,
> 'items_per_row': 6,
> 'margin': 0,
> 'image_size': 150
> });
> });
> }
> })(jQuery);
> ```

Unfortunately it doesn’t help.

Actually I use 
`If ($(“#instaFeed”).length>0){
//initialize 
}`

**Comment by blackseabreathe** [comment]
I use **instagramFeed** in every page of the site in `footer`.

**Comment by jsanahuja** [comment]
The new client-side caching (`cache_time`) feature implemented in #97 (2.0.0 release) should deal with this.
